src/automaton/core/node/node.cc

```diff
@@ -565,7 +562,7 @@ bool node::address_parser(const string& s, string* protocol, string* address) {
 }
 
 void node::on_message_received(peer_id c, std::shared_ptr<char> buffer, uint32_t bytes_read, uint32_t mid) {
-  LOG(DBUG) << "RECEIVED: " << core::io::bin2hex(string(buffer.get(), bytes_read)) << " from peer " << c;
+  // LOG(DBUG) << "RECEIVED: " << core::io::bin2hex(string(buffer.get(), bytes_read)) << " from peer " << c;
```

Comment:
I would make all of those LOG(INFO) and just suppress the INFO logs by default if it's too much output.